assets/src/application/ui/display/icons/svgs/style.scss

```diff
@@ -0,0 +1,4 @@
+.ee-svg {
+	display: inline-flex;
+	margin: 0 var(--ee-margin-micro);
```

Comment:
w/o that icons wasn't aligned:

![image](https://user-images.githubusercontent.com/1477453/80214901-47e04680-8644-11ea-88e3-ccce083cb92d.png)